.gitmodules

```diff
@@ -1,6 +1,3 @@
-[submodule "staging-areas"]
```

Comment:
Hey, @bbpennel, is this what you wanted? I had already rebased my branch off of fcrepo4, and I had added this to the .gitignore file, so I'm not sure why continues to show up uninvited.